socorro/unittest/cron/jobs/test_matviews.py

```diff
@@ -154,35 +153,6 @@ def mock_utc_now():
                 assert not information[app_name]['last_error']
                 assert information[app_name]['last_success']
 
-    def test_reports_clean_with_dependency(self):
-        config_manager = self._setup_config_manager(
-            'socorro.cron.jobs.matviews.DuplicatesCronApp|1h\n'
-            'socorro.cron.jobs.matviews.ReportsCleanCronApp|1h'
-        )
-
-        with config_manager.context() as config:
-            tab = CronTabberApp(config)
-            tab.run_all()
-
-            information = self._load_structure()
-            assert information['reports-clean']
-            assert not information['reports-clean']['last_error']
-            assert information['reports-clean']['last_success']
```

Comment:
I'm not sure what this was testing, but if I remove `DuplicatesCronApp` from the list, then it fails. I figured I'd just nix it. It kind of seems like it's testing something in the crontabber library. If that's true, then we don't need to test that here--the library tests can deal with that.